Lib/_colorize.py

```diff
@@ -62,6 +65,6 @@ def can_colorize() -> bool:
             return False
 
     try:
-        return os.isatty(sys.stderr.fileno())
+        return os.isatty(file.fileno())
     except io.UnsupportedOperation:
-        return sys.stderr.isatty()
+        return sys.stdout.isatty()
```

Comment:
Why using always sys.stdout?

```suggestion
        return file.isatty()
```